### DISCUSSION TITLE
Silent Output in GoogleTTS

### DISCUSSION THREAD

**Discussion by hafsalm** [fonoster/yaps#595] [Q&A]
Hi all,

I'm new to Fonoster and tried a simple GoogleTTS "hello world" project. 

The sln24 file is created, but there's no sound. Any ideas why? Thanks!

```TS
import {VoiceRequest, VoiceServer, VoiceResponse} from "@fonoster/voice";
import logger from "@fonoster/logger";
import GoogleTTS  from "@fonoster/googletts";

const speechConfig = { keyFilename: "./google.json" };

new VoiceServer(serverConfig).listen(
  async (req: VoiceRequest, res: VoiceResponse) => {
    logger.verbose(req);
    await res.answer();
    await res.use(new GoogleTTS(speechConfig));
    await res.say("Hello world"); 
    await res.hangup();
  }
);

```

<img width="539" alt="image" src="https://github.com/fonoster/fonoster/assets/3657496/a941c478-6431-4962-b26c-e843cf27475b">

**Comment by psanders** [comment]
Do you, by chance, have an M2?

**Reply by hafsalm** [reply]
I'm trying this on my M1. Are there any hardware requirements for the TTS?

**Reply by psanders** [reply]
I have observed this behavior at least once on my M2, so I am trying to identify a pattern.

**Reply by hafsalm** [reply]
I tried it on my i7 Windows machine as well. Same issue. Is there any way to solve this?
